SPECS-EXTENDED/webkit2gtk3/webkit2gtk3.spec

```diff
@@ -0,0 +1,705 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Name:           webkit2gtk3
+Version:        2.36.1
+Release:        1%{dist}
+Summary:        GTK Web content engine library
+
+License:        LGPLv2
+URL:            https://www.webkitgtk.org/
+Source0:        https://webkitgtk.org/releases/webkitgtk-%{version}.tar.xz
+
+BuildRequires:  bison
+BuildRequires:  ccache
+BuildRequires:  bubblewrap
+BuildRequires:  cmake
+BuildRequires:  flex
+BuildRequires:  gcc-c++
+BuildRequires:  gettext
+BuildRequires:  git
+BuildRequires:  gnupg2
+BuildRequires:  gperf
+BuildRequires:  gtk-doc
+BuildRequires:  hyphen-devel
+BuildRequires:  libatomic
+BuildRequires:  ninja-build
+BuildRequires:  perl(English)
+BuildRequires:  perl-local-lib
+BuildRequires:  perl-generators
+BuildRequires:  perl-Module-Build
+BuildRequires:  perl-File-Find-Rule
+BuildRequires:  perl(File::Copy::Recursive)
+BuildRequires:  perl(File::Path)
+BuildRequires:  perl(File::Spec)
+BuildRequires:  perl(FindBin)
+BuildRequires:  perl(JSON::PP)
+BuildRequires:  perl(Switch)
+BuildRequires:  python3
+BuildRequires:  ruby
+BuildRequires:  rubygems
+BuildRequires:  rubygem-json
+BuildRequires:  xdg-dbus-proxy
+
+BuildRequires:  pkgconfig(atspi-2)
+BuildRequires:  pkgconfig(cairo)
+BuildRequires:  pkgconfig(egl)
+BuildRequires:  pkgconfig(enchant-2)
+BuildRequires:  pkgconfig(fontconfig)
+BuildRequires:  pkgconfig(freetype2)
+BuildRequires:  pkgconfig(gl)
+BuildRequires:  pkgconfig(glib-2.0)
+BuildRequires:  pkgconfig(glesv2)
+BuildRequires:  pkgconfig(gobject-introspection-1.0)
+BuildRequires:  pkgconfig(gstreamer-1.0)
+BuildRequires:  pkgconfig(gstreamer-plugins-base-1.0)
+BuildRequires:  pkgconfig(gtk+-3.0)
+BuildRequires:  pkgconfig(harfbuzz)
+BuildRequires:  pkgconfig(icu-uc)
+BuildRequires:  pkgconfig(lcms2)
+BuildRequires:  pkgconfig(libgcrypt)
+BuildRequires:  pkgconfig(libjpeg)
+BuildRequires:  pkgconfig(libnotify)
+BuildRequires:  pkgconfig(libopenjp2)
+BuildRequires:  pkgconfig(libpcre)
+BuildRequires:  pkgconfig(libpng)
+BuildRequires:  pkgconfig(libseccomp)
+BuildRequires:  pkgconfig(libsecret-1)
+BuildRequires:  pkgconfig(libsoup-2.4)
+BuildRequires:  pkgconfig(libsystemd)
```

Comment:
Historically, we've run into issues where `pkgconfig(libsystemd)` will sometimes resolve to a `systemd-bootstrap` package instead of a package from the proper `systemd` package. While we come up with a long term solution for this issue, can you change this to just be a build requirement on `systemd`?

Reply:
Fixed.